test/integration/test_forward.rb

```diff
@@ -24,6 +24,18 @@
 class ForwardTestBase < NetSSHTest
   include IntegrationTestHelpers
 
+  # @yield [pid, port]
+  def start_sshd(port = '2200')
```

Comment:
what about naming it somehting like `start_sshd_7_or_later` so it's more descriptive?